moto/batch/models.py

```diff
@@ -282,6 +286,9 @@ def _get_resource_requirement(self, req_type: str, default: Any = None) -> Any:
         :return: The value of the resource requirement, or None.
         :rtype: any
         """
+        assert (
```

Comment:
If we want to validate this, we should throw the same exception that AWS throws. See the other `ClientExceptions` that are thrown in this file - that guarantees at least that the same format is in-line with AWS.

Reply:
removed, this was mostly for testing to make sure I didn't miss a spot